docs/development/contribution-documentation/documentation-style-guide/README.md

```diff
@@ -1,10 +1,4 @@
----
-sidebar_navigation:
-  title: Documentation style guide
-  priority: 700
-description: What styles and formats to follow when contributing to the OpenProject documentation
-keywords: documentation style guide, style guide, format, style
----
+
```

Comment:
The yaml header has been accidentally removed.

Reply:
Hi @as-op
did you fix that already? I think Typora default behaviour.
I do not see this written in the style guide, maybe somewhere else, we could raise the awareness for that at the documentation contributor...

Reply:
No, unfortunately I can't fix it here, it's in your fork.

Maybe it's a setting in Typora? So far this did not happen for other Typora users.